src/main/resources/db/migration/V1.8__Create_CoursePrice_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE course_price (
+price_id bigint not null auto_increment,
+price integer,
+course_id bigint,
+primary key (price_id)
+);
+alter table course_price add constraint FK4leaja1jtefsdfaiqqk4yuxna foreign key (course_id) references course (course_id);
```

Comment:
add one more line, please

Reply:
done

Reply:
What does FK4leaja1jtefsdfaiqqk4yuxna mean?

I suggest to rename it to COURSE_PRICE_COURSE_FK

Reply:
Fixed